### ISSUE TITLE
Proposals should be stored as an array

### ISSUE THREAD

**Issue by dorians** [Cryptonomic/Conseil#672]

**Comment by vishakh** [comment]
Could you please elaborate more?

**Comment by ivanopagano** [comment]
After a failed attempt to track down the exact compilation error with slick-pg array-enabled definitions, I'm switching to another task
I will resume exploration there when I have more time. What I have found, from a stack-trace that no longer appears on my branch (why not!?) is that the derivation of something gets stuck? The weird thing is that it pointed to the genSchema code, specifically to the derivation of pureconfig loader? Sounds weird to say the least, so I'm not 100% betting on that

In any case, even with a correct generator (which is feasible) the error is bound to the compilation of the generated code, which could be made manually with the same end-result. What I mean is that the fault is not related to the schema generator or the generated schema itself... it looks like the addition of a slick column that includes the array definition makes some part of the type inference loop...

I also played with the thought that it might be related to the additional features introduced by the custom postgres profile, which gets stuck on some implicit inference?

I only have guesses for the moment

I leave these notes for anyone willing to make investigations himself before I give it another shot
